app/models/foreman_xen/xenserver.rb

```diff
@@ -287,13 +287,14 @@ def create_vm_from_builtin(args)
       if args[:hypervisor_host] != ''
         host = client.hosts.find { |host| host.name == args[:hypervisor_host] }
         logger.info "create_vm_from_builtin: host : #{host.name}"
-      elsif host = client.hosts.first
+      else
```

Comment:
It is working but gives warning in logs. 
It definitely should be else :)

Example to prove it:

args = ''
if args != ''
	host = "defined"
elsif 
	host = "random"
end

[root@foreman-lab test]# ruby test1.rb
test1.rb:5: warning: found = in conditional, should be ==
random